react/package.json

```diff
@@ -4,7 +4,9 @@
   "private": true,
   "dependencies": {
     "@bugsnag/js": "^8.2.0",
+    "@types/geojson": "^7946.0.16",
     "axios": "^0.21.0",
+    "d3": "^7.9.0",
```

Comment:
this will pull d3 as a dependency into our js bundle, increasing the size for every page load. But we don't need that because we load it from an external source on the dashboard page, right? If we need it for types, I think adding it in dev dependencies should be enough.